lib/fluent/config/configure_proxy.rb

```diff
@@ -89,6 +95,9 @@ def merge(other) # self is base class, other is subclass
         if overwrite?(other, :alias)
           raise ConfigError, "BUG: subclass cannot overwrite base class's config_section: alias"
         end
+        if overwrite?(other, :configured_in_section)
+          raise ConfigError, "BUG: subclass cannot overwrite base class's config_section: configured_in"
```

Comment:
No. This message is for authors of plugins, and plugins use `configured_in` method to specify this value.